app/src/main/cpp/asurfacerenderer/asurface_jni.cpp

```
@@ -0,0 +1,164 @@
+        jlong ahbPtr, jint fenceFd, jlong windowId, jlong serial)
+{
+    if (auto* r = g_ctx.load(std::memory_order_acquire)) {
+        if (ahbPtr) {
+            r->setWindowBuffer(
+                    (int64_t) contentId,
+                    reinterpret_cast<AHardwareBuffer *>(ahbPtr),
+                    (int) fenceFd,
+                    (int64_t) windowId,
+                    (int64_t) serial);
+        }
+    }
```

Cppcheck (2.21.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)